src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -1077,6 +1077,15 @@ def blob_tier_validator(cmd, namespace):
         raise ValueError('Blob tier is only applicable to block or page blob.')
 
 
+def blob_rehydrate_priority_validator(namespace):
+    if namespace.blob_type == 'page' and namespace.rehydrate_priority:
+        raise ValueError('--rehydrate-priority is only applicable to block blob.')
+    if namespace.tier == 'Archive' and namespace.rehydrate_priority:
+        raise ValueError('--rehydrate-priority is only applicable to rehydrate blob data from the archive tier.')
+    if namespace.rehydrate_priority is None:
+        namespace.rehydrate_priority = 'Standard'
```

Comment:
According to the design here, you will provide default value for rehydrate_priority, right? Is it expected?

Reply:
Yes, that's how it's designed